ledger-tool/src/main.rs

```diff
@@ -602,6 +834,13 @@ fn main() {
             .about("Print the ledger")
             .arg(&starting_slot_arg)
         )
+        .subcommand(
+            SubCommand::with_name("print-csv")
```

Comment:
Could we get away with removing the "slot-csv" command if "print-csv" supported an --end-slot argument?